tests/tensor/nnet/test_basic.py

```diff
@@ -906,71 +895,6 @@ def validate_grad_graph(func):
             assert softmax_grad_legacy not in ops
 
 
-def test_argmax_pushdown():
```

Comment:
It should be possible to refactor these so that they work with each `Op` separately.